js/app.js

```diff
@@ -84,65 +84,26 @@ var seattleCenter = new CreateStore('Seattle Center', 11, 38, 3.7);
 var capitolHill = new CreateStore('Capitol Hill', 20, 38, 2.3);
 var alki = new CreateStore('Alki', 2, 16, 4.6);
 
+CreateStore.renderHrTotals = function() {
+  var tableElement = document.getElementById('table');
+  var tableRow = document.createElement('tr');
+  tableRow.setAttribute('class', 'hrTotals');
+  tableElement.appendChild(tableRow);
+  var totalTime = document.createElement('td');
+  totalTime.textContent = 'Hourly Totals';
+  tableRow.appendChild(totalTime);
+  for (var i in hrsOfOperation) {
+    tableElement = document.getElementById('table');
+    totalTime = document.createElement('td');
+    totalTime.textContent = hrlyTotals[i +15];
```

Comment:
One problem with your code right now: you're going to `i + 15` aka indices 15-29, so that's definitely an issue.